### PR TITLE
Update navigation to reflect recent changes

### PR DESCRIPTION
**Pull request by karelhala**
* Update all packages
 * Update index.html to reflect recent changes to navigation

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1 +1,4 @@
 @import './src/Utilities.scss';
+#root {
```
Comment (ryelo) [inline src/App.scss]: You won't need to set this if you use the appropriate `PageHeader` and `Main` Components

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,19 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
     <head>
         <meta charset="UTF-8">
-        <title>Red Hat Insights | Inventory</title>
+        <title>Red Hat Insights | Sample App</title>
```
Comment (ryelo) [inline src/index.html]: `Sample App` to `Inventory`

File: src/routes/Inventory.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ class Inventory extends Component {
                 <PageHeader>
                     <PageHeaderTitle title='Inventory'/>
                 </PageHeader>
-                <Section type='content'>
+                <Section className="pf-l-page__main-section">
```
Comment (ryelo) [inline src/routes/Inventory.js]: You can just use the `<Main>` component here

Comment (ryelo) [reply src/routes/Inventory.js]: See: https://github.com/RedHatInsights/insights-frontend-starter-app/blob/master/src/SmartComponents/SamplePage/SamplePage.js

Comment (karelhala) [reply src/routes/Inventory.js]: Oh I see!

**Pull request merged**